packages/components/psammead-section-label/src/index.jsx

```diff
@@ -57,11 +58,25 @@ const Wrapper = styled.div`
         ${({ script }) => (script ? top(script) : 'top: 0')};
       }
     `}
+
+  ${({ hideSectionHeader }) =>
+    hideSectionHeader &&
+    css`
+      @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+        clip-path: inset(100%);
```

Comment:
Would be nice to not have to redefine this twice in psammead, as its the same as the visually hidden component. As I dont see an easy way of doing without pulling it into a seperate package first, this looks fine for now.

If we use this in a 3rd place we should refactor this at that point :)